### DISCUSSION TITLE
Observability integration

### DISCUSSION THREAD

**Discussion by soyoh** [adonisjs/core#4685] [Help]
Hello,
I'd like to give Adonis a try for a project, and was wondering if there is any observability tools that can be easily integrated? For instance, DataDog or AppSignal?

I couldn't find anything in particular regarding this.
Any pointers would be appreciated
Thanks!

**Comment by pomarec** [comment]
Or Open Telemetry support (like: [@opentelemetry/instrumentation-nestjs-core](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-nestjs-core) )
